src/Symfony/Component/Translation/Loader/YamlFileLoader.php

```diff
@@ -54,7 +54,7 @@ public function load($resource, $locale, $domain = 'messages')
         try {
             $messages = $this->yamlParser->parse(file_get_contents($resource));
         } catch (ParseException $e) {
-            throw new InvalidResourceException(sprintf('Error parsing YAML, invalid file "%s"', $resource), 0, $e);
+            throw new InvalidResourceException(sprintf('Error parsing YAML, invalid file "%s" at line %d', $resource, $e->getParsedLine()), 0, $e);
```

Comment:
we should check line value `>= 0` see [ParseException.php#L136-L138](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Yaml/Exception/ParseException.php#L136-L138)  or even use ParseException :
```php
$parseException = new ParseException('Error parsing YAML', $e->getParsedLine(), $e->getSnippet(), $resource);
throw new InvalidResourceException($parseException->getMessage());
```